Src/FluentAssertions/Collections/StringCollectionAssertions.cs

```diff
@@ -242,5 +242,58 @@ private IEnumerable<string> AllThatMatch(string wildcardPattern)
                 }
             });
         }
+
+        /// <summary>
+        /// Asserts that the collection does not contain any string that matches a wildcard pattern.
+        /// </summary>
+        /// <param name="wildcardPattern">
+        /// The wildcard pattern with which the subject is matched, where * and ? have special meanings.
+        /// </param>
+        /// <param name="because">
+        /// A formatted phrase as is supported by <see cref="string.Format(string,object[])" /> explaining why the assertion
+        /// is needed. If the phrase does not start with the word <i>because</i>, it is prepended automatically.
+        /// </param>
+        /// <param name="becauseArgs">
+        /// Zero or more objects to format using the placeholders in <see cref="because" />.
+        /// </param>
+        public AndWhichConstraint<StringCollectionAssertions, string> NotContainMatch(string wildcardPattern, string because = "",
```

Comment:
Wondering whether a negative assertion should return an `AndWhichConstraint`...
I have an odd feeling about asserting that a collection does not contains something and then continue asserting on what *didn't* match.
Also because `.Which` will only(?) make sense if the collection has *exactly* one non-matching item, which can be returned from `.Which`.
Maybe I'm just struggling finding a good example.

Reply:
No, I think you are right. Let's just use an `AndConstraint`

Reply:
Just pushed this change :)